### PR TITLE
Prepare documentation to move to new docker images

### PR DESCRIPTION
**Pull request by GabLeRoux**
#### Changes

- Update actions versions
- Update links to docker images

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: docs/github/getting-started.md [hunk 1]
```diff
@@ -84,26 +82,26 @@ jobs:
...
-      - uses: actions/cache@v1.1.0
+      - uses: actions/cache@v2.1.3
         with:
           path: Library
           key: Library
 
       # Test
       - name: Run tests
-        uses: webbertakken/unity-test-runner@v1.3
+        uses: webbertakken/unity-test-runner@v2
```
Comment (davidmfinol) [inline docs/github/getting-started.md:92]: We should probably also change from `webbertakken` to `game-ci`.

File: docs/github/getting-started.md [hunk 2]
```diff
@@ -84,26 +82,26 @@ jobs:
...
         with:
-          unityVersion: 2019.2.11f1
+          unityVersion: 2019.4.17f1
```
Comment (davidmfinol) [inline docs/github/getting-started.md:94]: With v2, we can leave out unityVersion, as it now defaults to whatever is in their ProjectSettings/ProjectVersiontxt. I think we should encourage using this default.

File: docs/github/getting-started.md (hunk 1 above)
Comment (GabLeRoux) [reply docs/github/getting-started.md:92]: I don't know much about github actions. Renaming a github actions repo does rename the action itself?

File: docs/github/getting-started.md (hunk 2 above)
Comment (GabLeRoux) [reply docs/github/getting-started.md]: Removed it and mentioned it defaults to `ProjectSettings/ProjectVersion.txt` 👍

File: docs/github/getting-started.md (hunk 1 above)
Comment (webbertakken) [reply docs/github/getting-started.md:92]: Yea you can now use game-ci/action-name as they are moved to game-ci. I just published a new version of builder, which indeed referenced the same marketplace action.

Comment (webbertakken) [reply docs/github/getting-started.md:92]: image

File: docs/github/getting-started.md [hunk 3]
```diff
@@ -42,17 +42,15 @@ Read the official documentation on how to setup a
...
+#### Supported unity versions
+
+Unity Actions are using 
+[game-ci/docker](https://github.com/game-ci/docker/) 
+since `unity-builder` version 2. Any version in this 
+[list](https://hub.docker.com/r/unityci/editor/tags)
 can be used to test and build projects.
 
-It's generally considered good practice to use the same Unity version for Unity Actions as you do to develop your project.
+It's generally considered good practice to use the same Unity version for Unity Actions as you do to develop your project. No need to specify your unity version anywhere, `unity-builder` and `unity-test-runner` will use the one specified in `ProjectSettings/ProjectVersion.txt`.
```
Comment (webbertakken) [inline docs/github/getting-started.md:53]: I would keep it rather short, and only document this explanation under unityVersion. 

The philosophy should always be trying to become zero-config, but that shouldn't mean more documentation.

Comment (GabLeRoux) [reply docs/github/getting-started.md:53]: So we shouldn't even mention it's good practice, it does it by default now. I'll delete whole line. Good point 👍

Comment (webbertakken) [reply docs/github/getting-started.md]: Even better!

File: docs/github/getting-started.md (hunk 1 above)
Comment (GabLeRoux) [reply docs/github/getting-started.md]: This change probably needs to be done at a few other places, not only in getting-started like I did. 👍

**Comment by webbertakken** [comment]
We'll probably need to close this one, and apply these changes to v2 specifically, once we have docs versioning.

**Comment by davidmfinol** [comment]
Why not keep this open, and just wait to apply it until after v2 is stable?

**Comment by webbertakken** [comment]
We can see if the changes will still be compatible. They may not be.

**Comment by davidmfinol** [comment]
Superseded by https://github.com/game-ci/documentation/pull/99

**Pull request closed without merging**